### PR TITLE
New kwl features

### PR DESCRIPTION
**Pull request by kovacsbalu**

### CODE REVIEW

File: bin/eazyctrl [hunk 1]
```diff
@@ -60,11 +60,19 @@ _READ_VAR_TIMEOUT_RANGE = 0.1, 0.5
 
 # Dictionary of known air exchanger (KWL) features.
 _KWL_FEATURES = {
+    'party_mode': ("v00094", 1, int, str),
+    'stand_by_mode': ("v00099", 1, int, str),
```
Comment (baradi09) [inline bin/eazyctrl]: Let's use `standby_mode` instead.

File: bin/eazyctrl [hunk 2]
```diff
@@ -60,11 +60,19 @@ _READ_VAR_TIMEOUT_RANGE = 0.1, 0.5
...
+    'operating_mode': ("v00101", 1, int, str),
     'fan_stage': ("v00102", 1, int, str),
+    'percentage': ("v00103", 3, int, None),
```
Comment (baradi09) [inline bin/eazyctrl]: Probably `fan_stage_percentage` would be more clear.

File: bin/eazyctrl [hunk 3]
```diff
@@ -60,11 +60,19 @@ _READ_VAR_TIMEOUT_RANGE = 0.1, 0.5
...
     'temp_outside_air': ("v00104", 7, float, None),
     'temp_supply_air': ("v00105", 7, float, None),
     'temp_outgoing_air': ("v00106", 7, float, None),
     'temp_extract_air': ("v00107", 7, float, None),
+    'supply_air_rpm': ("v00348", 4, int, None),
+    'extract_air_rpm': ("v00349", 4, int, None),
+    'bypass': ("v02119", 1, int, None),
```
Comment (baradi09) [inline bin/eazyctrl:74]: Interesting, variable `v02119` does not appear in the manual I have found at the Helios page, but I have seen it being discussed in some forums. Can you point me by any chance to an official Helios document which contains that variable?

**Review by baradi09 — COMMENTED** [review]
Could you change those two field names and eventually squash your commits into a single one? Otherwise good to go. Köszi!

File: bin/eazyctrl (hunk 3 above)
Comment (kovacsbalu) [reply bin/eazyctrl:74]: I got this code from UI source. You can see codes when you sniff http packages.

**Comment by kovacsbalu** [comment]
Fixed. 
Szívesen :)

**Pull request merged**